komga-webui/src/components/dialogs/EditBooksDialog.vue

```diff
@@ -490,7 +491,7 @@ export default Vue.extend({
       deep: true,
       async handler(val: []) {
         const index = val.findIndex(x => x !== null)
-        this.authorSearchResults = await this.$komgaReferential.getAuthorsNames(val[index])
+        this.searchAuthors(val[index])
```

Comment:
The handler function can be replaced by this:
```ts
handler: debounce(async function (this: any, val: []) {
        const index = val.findIndex(x => x !== null)
        this.authorSearchResults = await this.$komgaReferential.getAuthorsNames(val[index])
      }, 500),
```

no need to define another function